apps/src/templates/BaseDialog.jsx

```diff
@@ -156,6 +149,16 @@ export default class BaseDialog extends React.Component {
       }),
       ...this.props.style,
     };
+    xCloseStyle = {
+      position: 'absolute',
+      top: 0,
+      right: 0,
+      padding: 0,
+      color: '#ddd',
```

Comment:
Is there a color variable that could be used here? `$neutral_dark30` ([ref](https://github.com/code-dot-org/code-dot-org/blob/dace2b40783cf58aaa97ded4254d93753f50bd83/shared/css/color.scss#L170)) is the right color.

Reply:
Good call thank you! Done!